src/Compilers/CSharp/Portable/Symbols/Source/SourceExtensionTypeSymbol.cs

```diff
@@ -231,6 +231,7 @@ private ExtensionInfo MakeDeclaredExtensionInfo(ConsList<TypeSymbol>? basesBeing
             SourceLocation? underlyingTypeLocation = null;
 
             Debug.Assert(basesBeingResolved == null || !basesBeingResolved.ContainsReference(this.OriginalDefinition));
```

Comment:
>Debug [](http://example.com/codeflow?start=12&length=5)

It doesn't look like there are any meaningful changes in this file. Consider reverting to its original state.